### DISCUSSION TITLE
Noob Doctor Needs Help wiht i9

### DISCUSSION THREAD

**Discussion by miltonport** [swar/Swar-Chia-Plot-Manager#1023] [Q&A]
Hey Guys.
What are the best settings to use Swar on my setup?
Really Appreciate it .

**I create 10 diffrent parallel task manually to 10 different HDD each time via chia gui in order the cut the copy time .
Can I configure Swar in a way that it would do this for me?**

**SSD:** RAID0   2X2TB Samsung 970 EVO PLUS 
**Ram :** GSKILL 64GB (2x32GB) 3200Mhz CL16 DDR4 
**Processor:** i9 10850K 3.60GHz 20MB 10 Core 20 Thread

**Comment by cherymann** [comment]
Hello, take a second and look at the config. You can add multiple destination drives. The plot manager will use them in a round robin fashion.
First plot first drive, second plot second drive, third plot first drive. The example I give here is for a setup with two destination drives.

**Reply by miltonport** [reply]
thanks mate !
